sdk/communication/communication-chat/review/communication-chat.api.md

```diff
@@ -86,6 +86,9 @@ export interface ChatMessage {
     deletedOn?: Date;
     editedOn?: Date;
     id: string;
+    properties?: {
```

Comment:
`properties` is a very generic term. Everything here is a JavaScript property by definition.
Have we considered other names?

Reply:
there were some back and forth discussions at the PM level among other names : tags, attributes, metadata. At the end it was settled down on properties (same term used by teams). Ash can provide more details if interested.